app/poros/i_scream_results.rb

```diff
@@ -0,0 +1,89 @@
+class IScreamResults 
+
+  def service
+    IScreamService.new
+  end
+
+  def game_params(game_name)
+    service.get_game_exact(game_name)
+  end
+
+  def create_game_objects(game_name)
```

Comment:
This method is the brains. It will take the API response and then generate all the game related objects `genre_objects(data, game)` will create genres from the API response info